packages/rebber/src/types/table.js

```diff
@@ -0,0 +1,31 @@
+/* Expose. */
+const one = require('../one')
+module.exports = table
+
+const defaultMacro = (ctx, node) => {
+  let j = 0
+  const parsed = []
+  node.children.map(n => parsed.push(one(ctx, n, j++, node)))
```

Comment:
I see. By the way if you need an index you won't reuse later on, remember that `map`'s callback is `(item, index, items) => {}`.

Which means you could drop the `j` in the parent scope and do something like: const parsed = node.children.map((n, j) => one(ctx, n, j, node))`